scootapi/server/setup.go

```diff
@@ -32,7 +34,9 @@ func makeServers(
 func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 	bag := ice.NewMagicBag()
 	bag.PutMany(
-		endpoints.MakeStatsReceiver,
+		func(scope endpoints.StatScope) stats.StatsReceiver {
```

Comment:
Twitter Obs only supports time series that are at the minute granularity.  There is no way to get smaller